generators/go/sdk/versions.yml

```diff
@@ -1,3 +1,11 @@
+- version: 0.33.0
+  changelogEntry:
+    - type: feat
+      summary: >-
+        Add support for the `inlinePathParameters` configuration option, which generates
+        path parameters in the generated request type (if any) instead of as separate
+        positional parameters.
```

Comment:
nit: can you add an example of the `generators.yml` including that configuration

Reply:
```yml generators.yml 
groups: 
  - go-sdk: 
       name: fernapi/fern-go-sdk
       version: 0.33.0
       config: 
          inlinePathParameters: true
```

Reply:
Good call, will add it to the earlier changelog entry I had for `inlineFileProperties`, too.